src/openrct2/CmdlineSprite.cpp

```diff
@@ -450,6 +458,155 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
         sprite_file_close();
         return 1;
     }
+    else if (_strcmpi(argv[0], "exportalldat") == 0)
+    {
+        if (argc < 3)
+        {
+            fprintf(stdout, "usage: sprite exportall <spritefile> <output directory>\n");
```

Comment:
Should this have read `exportalldat`?

Reply:
It should have, yes.

https://github.com/OpenRCT2/OpenRCT2/pull/11799